OpenOversight/tests/test_routes.py

```diff
@@ -870,3 +870,18 @@ def test_admin_can_add_new_unit(mockdata, client, session):
         unit = Unit.query.filter_by(
             descrip='Test').one()
         assert unit.department_id == department.id
+
+
+def test_user_can_change_dept_pref(mockdata, client, session):
+    with current_app.test_request_context():
+        login_user(client)
+
+        form = ChangeDefaultDepartmentForm(dept_pref=1)
+
+        rv = client.post(
+            url_for('auth.change_dept'),
+            data=form.data,
+            follow_redirects=True
+        )
+
+        assert 'Updated!' in rv.data
```

Comment:
Nice test! One though - we could also assert that the correct department is assigned - i.e.:

```
        user = User.query.filter_by(email='jen@example.org').one()
        assert user.dept_pref == test_department_id
```

where `test_department_id` is what you set the department ID to be, i.e. in the example 1.